guides/cancelOrders.md

```diff
@@ -7,40 +14,16 @@
 This method is used to invalidate the `orderNonce`. If multiple maker orders share the same nonce, they will all be cancelled and non-executable. An `orderNonce` is also invalidated once an order with that nonce is executed.
 
 ```ts
-import { LooksRare, ChainId } from "@looksrare/sdk-v2";
```

Comment:
Yep. We renamed the main export from `LooksRare` to `HypercertExchangeClient`. It's been like that since the start, but we didn't update the docs yet to reflect that.